pages/linux/gdisk.md

```diff
@@ -0,0 +1,28 @@
+# gdisk
+
+> GPT (GUID Partition Table) disk partitioning tool.
+> More information: <https://manned.org/gdisk>.
+
+- Start partition manipulator for a specific disk:
+
+`sudo gdisk {{/dev/sdX}}`
+
+- Display partition table without entering interactive mode:
+
+`sudo gdisk -l {{/dev/sdX}}`
+
+- Backup partition table to a file before making changes:
+
+`sudo gdisk -b {{path/to/backup_file}} {{/dev/sdX}}`
+
+- Restore partition table from backup file:
+
+`sudo gdisk -l {{path/to/backup_file}} {{/dev/sdX}}`
+
+- Convert MBR partition table to GPT without data loss:
+
+`sudo gdisk -m {{/dev/sdX}}`
+
+- Display detailed help information:
+
+`sudo gdisk -? {{/dev/sdX}}`
```

Comment:
This does not exist. Are you using AI?

Reply:
I can't say if this is AI or not, but it looks like indeed this these commands haven't been tested.

`?` is an interactive command available in the interactive prompt of gdisk, but not an actual command-line argument. For example, you could do:

`sudo gdisk /path/to/device`

Which launches gdisk, which you can then do `?` on its own for help in the gdisk prompt.

However, there is no argument called `?`, you can't do `-?` in the command-line arguments before launching gdisk afaik.